sympy/functions/elementary/tests/test_integers.py

```diff
@@ -390,6 +395,10 @@ def test_ceiling():
     assert (ceiling(y) < n) == (y <= n - 1)
     assert (ceiling(y) > n) == (y > n)
 
+    assert ceiling(RootOf(x**5-x**2+1,0)) == 0
```

Comment:
There should also be a test for `floor`. It would better to pick an example where the result is something other than zero.